x-pack/plugins/enterprise_search/server/lib/check_access.ts

```diff
@@ -38,20 +40,53 @@ const DENY_ALL_PLUGINS = {
 export const checkAccess = async ({
   config,
   security,
+  spaces,
   request,
   log,
 }: ICheckAccess): Promise<IAccess> => {
```

Comment:
Super sorry for the extra hassle, we just merged in a typescript refactor where we removed the `I` prefixes from most of our interfaces - that's probably going to give you a merge conflict here, the fix should be this:

```
}: CheckAccess): Promise<Access> => {
```